### PR TITLE
Fix validation summary ordering

### PR DESCRIPTION
**Pull request by stevospinks**

### CODE REVIEW

File: src/EA.Weee.Web/Areas/Admin/ViewModels/Aatf/FacilityViewModelBase.cs [hunk 1]
```diff
@@ -44,21 +37,44 @@ public abstract class FacilityViewModelBase : IValidatableObject
...
+
+        [Required]
+        [Display(Name = "Compliance year")]
+        public short ComplianceYear { get; set; }
+
+        public FacilityType FacilityType { get; set; }
+
+        public bool ModelValidated { get; private set; }
+
+        public static IEnumerable<string> ValidationMessageDisplayOrder => new List<string>
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web/Areas/Admin/ViewModels/Aatf/FacilityViewModelBase.cs]: Can we add a unit test to check this property and that they are in this order :)

File: src/EA.Weee.Web/Areas/Admin/ViewModels/AddAatf/AddFacilityViewModelBase.cs [hunk 2]
```diff
@@ -15,6 +15,34 @@ public abstract class AddFacilityViewModelBase : FacilityViewModelBase
 
         public IEnumerable<short> ComplianceYearList => new List<short> { 2019 };
 
+        public static new IEnumerable<string> ValidationMessageDisplayOrder => new List<string>
+        {
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web/Areas/Admin/ViewModels/AddAatf/AddFacilityViewModelBase.cs]: Can we add a unit test to check this property and that they are in this order :)

File: src/EA.Weee.Web/Areas/Admin/ViewModels/Aatf/FacilityViewModelBase.cs (hunk 1 above)
Comment (stevospinks) [reply src/EA.Weee.Web/Areas/Admin/ViewModels/Aatf/FacilityViewModelBase.cs]: Added

File: src/EA.Weee.Web/Areas/Admin/ViewModels/AddAatf/AddFacilityViewModelBase.cs (hunk 2 above)
Comment (stevospinks) [reply src/EA.Weee.Web/Areas/Admin/ViewModels/AddAatf/AddFacilityViewModelBase.cs]: Added

**Pull request merged**